client/components/app-promo/index.jsx

```diff
@@ -88,14 +88,10 @@ export const AppPromo = React.createClass( {
 
 AppPromo.defaultProps = {
 	promoItem: getRandomPromo(),
-	getPromoLink: ( location, promoCode ) => {
-		return getPromoLink( location, promoCode );
-	},
+	getPromoLink,
```

Comment:
I should have mentioned, but our style is to put these at the end of the list of options so that the key/value pairs come first and then the shorthand key/values combos